static/app/components/feedback/list/useMailboxCounts.tsx

```diff
@@ -0,0 +1,59 @@
+import {useMemo} from 'react';
+
+import type {Organization} from 'sentry/types';
+import {useApiQuery} from 'sentry/utils/queryClient';
+import {decodeList, decodeScalar} from 'sentry/utils/queryString';
+import useLocationQuery from 'sentry/utils/url/useLocationQuery';
+
+interface Props {
+  organization: Organization;
+}
+
+const MAILBOX = {
+  unresolved: 'issue.category:feedback is:unassigned is:unresolved',
+  resolved: 'issue.category:feedback is:unassigned is:resolved',
+  ignored: 'issue.category:feedback is:unassigned is:ignored',
+};
+
+export default function useMailboxCounts({organization}: Props) {
+  const queryView = useLocationQuery({
+    fields: {
+      end: decodeScalar,
+      environment: decodeList,
+      field: decodeList,
+      project: decodeList,
+      query: Object.values(MAILBOX),
+      queryReferrer: 'feedback_list_page',
+      start: decodeScalar,
+      statsPeriod: decodeScalar,
+      utc: decodeScalar,
+    },
+  });
+
+  const {data = {}, ...result} = useApiQuery(
```

Comment:
just curious - why is it `{data = {}, ...result}` instead of just `{data}`?

Reply:
i wanted to return the rest of the result info, like `isFetching` and so on, so that's why `...result` is there.

But it's kinda weird written this way. I changed it up to be `const result = useApiQuery(...)` and added a bunch of types to try to describe that conversion a bit more.